aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs

```diff
@@ -40,13 +40,13 @@ public class CanvasManager : MonoBehaviour
     /// Name of the start screen.
     /// </summary>
     [SerializeField]
-    private string _sceneNameStartingMenu = "Settings"; // TODO:: Load main start screen
+    private readonly string _sceneNameStartingMenu = "Settings";
```

Comment:
The SerializeField is currently useless. If you don't want to make it visible in the inspector, remove the [SerializeField]. If you DO want to make it visible in the inspector (so that you can easily adjust the strings) it should not be readonly.